src/libraries/Native/Unix/System.Security.Cryptography.Native.Android/pal_jni.h

```diff
@@ -236,10 +323,12 @@ void SaveTo(uint8_t* src, uint8_t** dst, size_t len, bool overwrite);
 jobject ToGRef(JNIEnv *env, jobject lref);
 jobject AddGRef(JNIEnv *env, jobject gref);
 void ReleaseGRef(JNIEnv *env, jobject gref);
+void ReleaseLRef(JNIEnv *env, jobject lref);
 jclass GetClassGRef(JNIEnv *env, const char* name);
 bool CheckJNIExceptions(JNIEnv* env);
 void AssertOnJNIExceptions(JNIEnv* env);
 jmethodID GetMethod(JNIEnv *env, bool isStatic, jclass klass, const char* name, const char* sig);
+jmethodID TryGetMethod(JNIEnv *env, bool isStatic, jclass klass, const char* name, const char* sig);
```

Comment:
In .NET Try parlance this isn't really Try (doesn't return a boolean).  GetOptionalMethod, perhaps?

If it's a reasonable "Try" from common JNI perspective, then OK.